src/RandomMapGenerator.py

```diff
@@ -200,7 +200,7 @@ def outputMap(
 ) -> None:
     if not os.path.exists(file_path):
         os.makedirs(file_path)
-    with open(f"{file_path}{file_name}.map", "w") as f:
+    with open(f"{file_path}/{file_name}.map", "w") as f:
```

Comment:
`os.path.join()`で統一したい
```suggestion
    with open(os.path.join(file_path,f"{file_name}.map"), "w") as f:
```

Reply:
確かに!
修正します.

Reply:
アルゴリズムの関係上, このままの実装だと, argで渡された値と若干の乖離があるのは避けられないことなので諦めます.
ただ, `argで渡された値/4`は簡単に実装できると思うのでやってしまいます.
(追加する処理としては, `argで渡された値/4`で端数が出た場合は切り捨てして小さいランダムマップの生成処理をする)

Reply:
修正しました!
[コメントで指摘された点の修正](https://github.com/KPC-U16/MapGenerator/pull/23/commits/a80898f80121f3a83d9476b96bc04ff9375b066f)